pkg/minikube/translate/translate.go

```diff
@@ -62,20 +60,11 @@ func T(s string) string {
 
 // DetermineLocale finds the system locale and sets the preferred language for output appropriately.
 func DetermineLocale() {
-	var locale string
-	// Allow windows users to overload the same env vars as unix users
-	if runtime.GOOS == "windows" {
-		locale = os.Getenv("LC_ALL")
```

Comment:
can u verify after this PR LC_ALL env variable can still override the language ?

Reply:
The comment in the library says
`// LC_ALL will overwrite all LC_* options.`

Reply:
lets paste the output of LC_ALL=fr minikube profile list and verify

Reply:
```
$ echo $LC_ALL
fr
$ ./out/minikube profile list

🤹  Fermeture en raison de MK_USAGE_NO_PROFILE : Aucun profil minikube n'a été trouvé.
💡  Suggestion : 

    Vous pouvez en créer un en utilisant 'minikube start'.
```